### PR TITLE
Remove post style that should be in core

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR removes some post styles moved in udata.

Connects to opendatateam/udata#1224

### CODE REVIEW

File: theme/less/gouvfr-post.less [hunk 1]
```diff
@@ -1,91 +1,6 @@
...
-            img {
-                background-color: white;
-                border: 1px solid @gray-lighter;
-            }
-        }
-    }
-
-    .history-section {
-        min-height: 200px;
+    .nav-section {
```
Comment (oncletom) [inline theme/less/gouvfr-post.less]: What is this addition related to? I was expecting to see the entire stylesheet to be gone.

Comment (noirbizarre) [reply theme/less/gouvfr-post.less]: This color is specific to the `gouvfr` theme, all others statement defines the layout and moved to the core.
